telegram/request/_requestdata.py

```diff
@@ -64,6 +64,11 @@ def parameters(self) -> Dict[str, Union[str, int, List, Dict]]:
     def json_parameters(self) -> Dict[str, str]:
         """Gives the parameters as mapping of parameter name to the respective JSON encoded
         value.
+
+        Tip:
+            By default, this property uses the standard libraries :func:`json.dumps`.
+            To use a custom library for JSON encoding, you can directly encode the keys of
+            :attr:`parameters` - note that string valued keys should not be JSON encoded.
```

Comment:
ah clever, so instead of making `RequestParameter` public, just tell the users to encode `self.parameters` 👍🏽